### PR TITLE
feat(anvil): add `anvil_getIntervalMining` API

### PR DESCRIPTION
**Pull request by feynman-x**
## Motivation

Closes https://github.com/foundry-rs/foundry/issues/8813 

## Solution

### CODE REVIEW

**Comment by feynman-x** [comment]
@grandizzy Followed your suggestion and created this new PR (old #9090). Please check if there is anything to improve.

**Comment by grandizzy** [comment]
> @grandizzy Followed your suggestion and created this new PR (old #9090). Please check if there is anything to improve.

Looks good, thank you! Just a small nit, maybe we should be consistent naming at par with `anvil_setIntervalMininig` e.g. `anvil_getIntervalMining` or `anvil_getMiningInterval`

File: crates/anvil/src/eth/api.rs
Comment (grandizzy) [inline crates/anvil/src/eth/api.rs:1669]:
```suggestion
    /// Returns the value of mining interval, if set.
```

File: crates/anvil/src/eth/api.rs
Comment (grandizzy) [inline crates/anvil/src/eth/api.rs:1671]:
```suggestion
    /// Handler for ETH RPC call: `anvil_getIntervalMining`.
```

File: crates/anvil/core/src/eth/mod.rs
Comment (grandizzy) [inline crates/anvil/core/src/eth/mod.rs:398]:
```suggestion
    #[cfg_attr(feature = "serde", serde(rename = "anvil_getIntervalMining", with = "empty_params"))]
```

**Review by grandizzy — COMMENTED** [review]
Looks good, thank you! Just a small nit, IMO we should be consistent naming at par with `anvil_setIntervalMining` e.g. `anvil_getIntervalMining` (instead `anvil_getIntervalMine`)

**Comment by feynman-x** [comment]
@grandizzy  Yes, it's reasonable. The PR has been updated.

**Review by grandizzy — APPROVED** [review]
Lgtm, thank you! Pending additional review before merging.

**Comment by feynman-x** [comment]
Thanks for your guidance. I'm so excited, this is my first PR for Foundry. Hope to contribute more in the future.

**Review by mattsse — APPROVED** [review]
great, this is a lot simpler

File: crates/anvil/src/eth/api.rs
Comment (holic) [inline crates/anvil/src/eth/api.rs:1672]: should this be `anvil_get_interval_mining` not `anvil_get_interval_ming`?

Comment (grandizzy) [reply crates/anvil/src/eth/api.rs:1672]: thank you! yes indeed, missed the typo :grin: preparing an update in https://github.com/foundry-rs/foundry/pull/9292

**Pull request merged**
